### PR TITLE
JBrowse: Added option to use locally cached genomes as reference fasta

### PR DESCRIPTION
**Pull request by Slugger70**

### CODE REVIEW

File: tools/jbrowse/jbrowse.xml [hunk 1]
```diff
@@ -216,13 +220,41 @@ $trackxml;
...
+        <when value="history">
+            <param 
+              format="fasta"
+              label="Select the reference genome"
+              name="genomes"
+              type="data"
+              multiple="True" />
+        </when>
+    </conditional>
+    <!-- <param label="Fasta Sequence(s)"
```
Comment (erasche) [inline tools/jbrowse/jbrowse.xml]: delete these lines

File: tools/jbrowse/jbrowse.xml [hunk 2]
```diff
@@ -216,13 +220,41 @@ $trackxml;
...
+        <when value="indexed">
+            <param 
+              help="If your genome of interest is not listed, contact the Galaxy team"
+              label="Select a reference genome"
+              name="genomes"
+              type="select"
+              >
+                <options from_data_table="all_fasta">
+                    <filter column="2" type="sort_by" />
+                    <validator message="No genomes are available for the selected input dataset" type="no_options" />
```
Comment (erasche) [inline tools/jbrowse/jbrowse.xml]: Is it possible for this to be a multi-select? Is there value in that?

Comment (Slugger70) [inline tools/jbrowse/jbrowse.xml]: No, type=select is not iterable

**Comment by bgruening** [comment]
@erasche  Ready to merge?

**Comment by yhoogstrate** [comment]
I think some semicolons have to be replaced with `&&` according to our guidelines. Also some paths have to be escaped with quotes.

**Comment by erasche** [comment]
@yhoogstrate paths here are all used inside XML so escaping shouldn't be necessary.

As for `;` -> `&` I'll get those in another commit, no reason for us to ask @Slugger70 do it.

**Comment by erasche** [comment]
@yhoogstrate https://github.com/galaxyproject/tools-iuc/pull/864

**Comment by bgruening** [comment]
Thanks a lot @Slugger70!

**Pull request merged**